xscen/aggregate.py

```diff
@@ -260,7 +532,7 @@ def compute_deltas(
     if "time" in ds:
         if (len(ds.time) >= 3) and (xr.infer_freq(ds.time) == "D"):
             raise NotImplementedError(
-                "xs.climatological_mean does not currently support daily data."
+                "xs.climatological_op does not currently support daily data."
```

Comment:
I think that this was actually a bad copy-paste...
```suggestion
                "xs.compute_deltas does not currently support daily data."
```